metadata-service/restli-client/src/main/java/com/linkedin/entity/client/JavaEntityClient.java

```diff
@@ -481,6 +483,12 @@ public void producePlatformEvent(
         _eventProducer.producePlatformEvent(name, key, event);
     }
 
+    @SneakyThrows
+    @Override
+    public void rollbackIngestion(@Nonnull String runId, @Nonnull Authentication authentication) throws Exception {
+        throw new MethodNotSupportedException();
```

Comment:
A couple options - 

1. we can use the same impl as RestliEntityClient so that we can actually use this method
2. we can embed a RestliEntityClient inside the JavaEntityClient and delegate to that when this is invoked. Just until we refactor some of the rollback endpoint code!

Reply:
I messages on slack but I think 2 is going to be our easiest path forward for now while also maintaining working methods in both clients! it was a bit tougher to `sendClientRequest` from the JavaEntityClient as it doesn't inherit from `BaseClient`